_events/47.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Beyond the Startup model: Building a coop - With Magdalena Ziomek"
+date: 2020-10-26 19:00:00 +0200
```

Comment:
```suggestion
date: 2020-10-26 19:00:00 +0100
```

The time zone changes on October 25th, I'll also see if there's way to omit these offsets in future, because it's annoying to have to remember this stuff..